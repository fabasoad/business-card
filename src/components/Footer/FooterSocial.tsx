import * as React from 'react'

export default function FooterSocial() {
  return (
    <div className="social d-flex justify-content-center">
      <ul className="icon-list">
        <li><a className="btn" target="_blank" rel="noopener noreferrer" href="https://twitter.com/fabasoad"><i className="fab fa-twitter fa-2x"></i></a></li>
        <li><a className="btn" target="_blank" rel="noopener noreferrer" href="https://github.com/fabasoad"><i className="fab fa-github fa-2x"></i></a></li>
        <li><a className="btn" target="_blank" rel="noopener noreferrer" href="https://www.linkedin.com/in/yevhenfabizhevskyi/"><i className="fab fa-linkedin fa-2x"></i></a></li>
        <li><a className="btn" target="_blank" rel="noopener noreferrer" href="https://stackoverflow.com/users/470214/fabasoad"><i className="fab fa-stack-overflow fa-2x"></i></a></li>
        <li><a className="btn" target="_blank" rel="noopener noreferrer" href="https://dev.to/fabasoad"><i className="fab fa-dev fa-2x"></i></a></li>
      </ul>
    </div>
  )
}
