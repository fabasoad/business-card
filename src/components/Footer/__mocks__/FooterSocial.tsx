import * as React from 'react'

export default function FooterSocial() {
  return (
    <div data-testid="FooterSocial"></div>
  )
}
