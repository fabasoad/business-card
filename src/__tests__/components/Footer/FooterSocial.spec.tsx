/// <reference types="jest" />
import * as React from 'react'
import { shallow } from 'enzyme'
import FooterSocial from '../../../components/Footer/FooterSocial'

test('should render FooterSocial correctly', () =>
  expect(shallow(<FooterSocial />)).toMatchSnapshot()
)
