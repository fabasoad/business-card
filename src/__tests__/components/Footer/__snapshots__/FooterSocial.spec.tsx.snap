// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render FooterSocial correctly 1`] = `
<div
  className="social d-flex justify-content-center"
>
  <ul
    className="icon-list"
  >
    <li>
      <a
        className="btn"
        href="https://twitter.com/fabasoad"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-twitter fa-2x"
        />
      </a>
    </li>
    <li>
      <a
        className="btn"
        href="https://github.com/fabasoad"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-github fa-2x"
        />
      </a>
    </li>
    <li>
      <a
        className="btn"
        href="https://www.linkedin.com/in/yevhenfabizhevskyi/"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-linkedin fa-2x"
        />
      </a>
    </li>
    <li>
      <a
        className="btn"
        href="https://stackoverflow.com/users/470214/fabasoad"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-stack-overflow fa-2x"
        />
      </a>
    </li>
    <li>
      <a
        className="btn"
        href="https://dev.to/fabasoad"
        rel="noopener noreferrer"
        target="_blank"
      >
        <i
          className="fab fa-dev fa-2x"
        />
      </a>
    </li>
  </ul>
</div>
`;
